product-page/src/features/products/ProductList.css

```diff
@@ -0,0 +1,90 @@
+.product-list-container {
+    max-width: 800px;
+    margin: 0 auto;
+    padding: 20px;
+    text-align: center;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+}
+
+.product-list-container h2 {
+    font-size: 24px;
+    margin-bottom: 20px;
```

Comment:
px is kind of an older way to define spacing in css, read this for more info https://engageinteractive.co.uk/blog/em-vs-rem-vs-px